tests/keras/backend/backend_test.py

```diff
@@ -300,8 +300,8 @@ def test_repeat_elements(self):
     def test_tile(self):
         shape = (3, 4)
         arr = np.arange(np.prod(shape)).reshape(shape)
-        check_single_tensor_operation('tile', arr, BACKENDS, n=[2, 1])
-        check_single_tensor_operation('tile', (2, 5), BACKENDS, n=[5, 2])
+        check_single_tensor_operation('tile', arr, WITH_NP, n=[2, 1])
```

Comment:
#11205 already adds the `tile` op. So I don't think it's necessary to add it in this PR.

Reply:
I removed my version